modules/mbedtls/crypto_mbedtls.cpp

```diff
@@ -30,25 +30,25 @@
 
 #include "crypto_mbedtls.h"
 
-#include "core/io/file_access.h"
-
 #include "core/config/engine.h"
 #include "core/config/project_settings.h"
 #include "core/io/certs_compressed.gen.h"
 #include "core/io/compression.h"
+#include "core/io/file_access.h"
 #include "core/os/os.h"
 
 #ifdef TOOLS_ENABLED
 #include "editor/editor_settings.h"
 #endif
-#define PEM_BEGIN_CRT "-----BEGIN CERTIFICATE-----\n"
-#define PEM_END_CRT "-----END CERTIFICATE-----\n"
-#define PEM_MIN_SIZE 54
 
 #include <mbedtls/debug.h>
 #include <mbedtls/md.h>
 #include <mbedtls/pem.h>
 
+#define PEM_BEGIN_CRT "-----BEGIN CERTIFICATE-----\n"
+#define PEM_END_CRT "-----END CERTIFICATE-----\n"
+#define PEM_MIN_SIZE 54
+
```

Comment:
I'm not sure if I'm reading it right, but I think at least the first two defines are also being defined somewhere in mbedtls codebase. Not sure if the order change would affect that (though realistically the value is the same anyway).